redist-installer.csproj : error NU1101: Unable to find package microsoft.netcore.app.crossgen2.linux-s390x.

Our CI cross-builds that build for ppc64le and s390x are failing recently with:
```
 /workdir/dotnet/src/sdk/src/Installer/redist-installer/redist-installer.csproj : error NU1101: Unable to find package microsoft.netcore.app.crossgen2.linux-s390x. No packages exist with this id in source(s): dotnet-eng, dotnet-libraries, dotnet-libraries-transport, dotnet-public, dotnet-tools, dotnet-tools-transport, dotnet10, dotnet10-transport, dotnet6, dotnet6-transport, dotnet7, dotnet7-transport, dotnet8, dotnet8-transport, dotnet8-workloads, dotnet9, dotnet9-transport, richnav, vs-impl. PackageSourceMapping is enabled, the following source(s) were not considered: prebuilt, previously-source-built, reference-packages, source-built-aspire, source-built-aspnetcore, source-built-command-line-api, source-built-deployment-tools, source-built-diagnostics, source-built-emsdk, source-built-fsharp, source-built-msbuild, source-built-nuget-client, source-built-roslyn, source-built-roslyn-analyzers, source-built-runtime, source-built-source-build-externals, source-built-sourcelink, source-built-symreader, source-built-templating, source-built-transport-arcade, source-built-transport-aspnetcore, source-built-transport-cecil, source-built-transport-emsdk, source-built-transport-msbuild, source-built-transport-razor, source-built-transport-roslyn, source-built-transport-runtime, source-built-transport-xdt, source-built-vstest. [/workdir/dotnet/src/sdk/source-build.slnf]
```

The ppc64le and s390x don't support crossgen2, so the build should probably not be trying to restore them.

cc @ViktorHofer

--- a/src/Installer/redist-installer/Directory.Build.targets
+++ b/src/Installer/redist-installer/Directory.Build.targets
@@ -14,7 +14,9 @@
   <Import Project="targets\BundledManifests.targets" />
   <Import Project="targets\BundledDotnetTools.targets" />
   <Import Project="targets\GenerateBundledVersions.targets" />
-  <Import Project="targets\Crossgen.targets" />
+  <!-- Crossgen is currently not supported on the s390x, ppc64le architecture as using mono instead of CoreCLR. -->
+  <Import Project="targets\Crossgen.targets"
+          Condition="'$(DISABLE_CROSSGEN)' == '' AND '$(Architecture)' != 's390x' AND '$(Architecture)' != 'ppc64le'" />
   <Import Project="targets\GenerateLayout.targets" />
   <Import Project="targets\GenerateArchives.targets" Condition="'$(PackInstaller)' != 'false'"/>
   <Import Project="targets\GenerateMSIs.targets" />

--- a/src/Installer/redist-installer/targets/Crossgen.targets
+++ b/src/Installer/redist-installer/targets/Crossgen.targets
@@ -12,9 +12,7 @@
     <PackageDownload Include="$(RuntimeNETCrossgenPackageName)" Version="[$(MicrosoftNETCoreAppRuntimePackageVersion)]" />
   </ItemGroup>
 
-  <!-- Crossgen is currently not supported on the s390x, ppc64le architecture as using mono instead of CoreCLR. -->
-  <Target Name="CrossgenLayout"
-          Condition="'$(DISABLE_CROSSGEN)' == '' AND '$(Architecture)' != 's390x' AND '$(Architecture)' != 'ppc64le'">
+  <Target Name="CrossgenLayout">
     <PropertyGroup>
       <CrossgenPath>$(NuGetPackageRoot)$(RuntimeNETCrossgenPackageName)/$(MicrosoftNETCoreAppRuntimePackageVersion)/tools/crossgen2$(ExeExtension)</CrossgenPath>
       <CreateCrossgenSymbols Condition="'$(CreateCrossgenSymbols)' == ''">true</CreateCrossgenSymbols>

